configure_build_serve/run.py

```diff
@@ -33,6 +34,7 @@ class Config(BaseSettings):
     host: str = "localhost"
     port: int = 8080
     client_url: str = "https://data.staging.ghga.dev/"
+    ribbon_text: str = "Development v$v"
```

Comment:
```suggestion
    banner_text: str = "Development v$v"
```

The variable used in the version banner is REACT_APP_BANNER_TEXT, not RIBBON_TEXT